docs/views.rst

```diff
@@ -73,11 +70,11 @@ The Universe view zooms all the way out to the extent of the observed universe:
 
 .. image:: images/universe.png
 
-.. note:: If you're ever lost inside a mode, the ``reset_view`` method 
-          backtracks to the mode's initial state and allows you to start your
-          exploration over again.
+.. note:: If you're ever lost inside a mode, backtrack to its initial state by
+          using the :meth:`~pywwt.BaseWWTWidget.reset_view` method.
 
 Panorama view
 -------------
+**Keep or no?**
```

Comment:
Is there any harm in keeping this for now?

Reply:
I don't think so. My knowledge of it is just incomplete because I'm not sure what the mode-specific controls are.